packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -20,6 +33,36 @@ def _ensure_routable():
     return _ensure_routable
 
 
+@retrying.retry(wait_fixed=2000,
```

Comment:
Yeah, the loops are very lightweight, so it's fine to retry the entire loop, rather than just the individual _check_overlay.